### DISCUSSION TITLE
An idea for managing page forms

### DISCUSSION THREAD

**Discussion by zhoujiahua** [wg-easy/wg-easy#456] [Ideas]
**Hello everyone,**

As shown in the picture are some of my ideas for the admin page.
_**Please let me know if you have better suggestions, thanks.**_

![](https://cdn.jsdelivr.net/gh/zhoujiahua/picture/2023/wg-easy-client-table.png)

**Comment by pheiduck** [comment]
I do not really like the design but it would be nice if we have some of the metrics in the Dashboard.
Feel free to create PR. :)

**Reply by peterlewis** [reply]
Agree with @pheiduck. To further add that the current clean interface is—to me—a positive. I'd be keen to ensure that any additions to the dashboard do not result in "clutter".

**Reply by zhoujiahua** [reply]
**Hi @pheiduck and @peterlewis,**

Thanks for your feedback! I'm glad to hear that you appreciate the current clean interface. I completely understand your concern about avoiding clutter, and I'll make sure that any additions to the dashboard are thoughtfully integrated to maintain the overall cleanliness and user-friendly experience.

I'll work on refining the design to incorporate the requested metrics without compromising the clean aesthetic. Once I have a proposed update, I'll create a pull request for further discussion.

Feel free to share any specific preferences or considerations you have in mind, and I'll do my best to incorporate them.

Thanks again for your valuable input!

**Best regards,**
Jerry Zhou
